Makefile

```diff
@@ -183,4 +184,7 @@ clean:
 deploy_kind:
 	# FIXME: This assumes kind is in cwd; intended to work with our other CI scripts and
 	# is a bit wonky for general use.
-	./kind load docker-image $(REGISTRY)/$(TARGET):$(IMAGE_VERSION) || true
\ No newline at end of file
+	./kind load docker-image $(REGISTRY)/$(TARGET):$(IMAGE_VERSION) || true
+
+brew:
```

Comment:
w.r.t. naming I think that I would prefer this to just be something like `native`.

I think it's perfect that you extracted/reused the logic in here and the `build_sonobuoy` target and this is the "right" way to build it locally if you have your repo set up in the appropriate manner. I have recently just run `go build` for my own little tests but using this target would be better and ensure my builds have version info on them and such. 

All of that is to say, even though `brew` was the initial motivation, it has more generic use cases and I think the name should reflect that.

Maybe:
 - `native`
 - `local`
 - another idea?

Reply:
When first making the target, I thought there might be something a bit more specific to be done for homebrew. You're right though, it would be better to have a more generic name. I'll change it to `native`.